src/css-reset.module.css

```diff
@@ -1,20 +1,17 @@
-@import 'theme.css';
-
-html {
-  box-sizing: border-box;
-}
-
-*, *:before, *:after {
-  box-sizing: inherit;
-}
-
-body {
+.ventura {
   font-family: var(--font-family);
   font-size: var(--font-size);
   color: var(--font-color);
+  box-sizing: border-box;
+}
+
+.ventura *,
+.ventura *:before,
+.ventura *:after {
+  box-sizing: border-box;
 }
 
-svg[viewBox="0 0 24 24"][stroke-linejoin="round"][stroke-linecap="round"] {
+svg[viewBox='0 0 24 24'][stroke-linejoin='round'][stroke-linecap='round'] {
```

Comment:
This SVG style will still only get applied if another component gets imported. These should be imported in index.ts